script.js

```diff
@@ -184,7 +194,7 @@ async function speak(text, language) {
 function speakSentence(text, language) {
 	var utterance = new SpeechSynthesisUtterance();
   utterance.text = text + '.';
-  utterance.lang = language || 'en';
+  utterance.lang = language || state.lang.speachTag;
```

Comment:
This was an ugly hack needed because the welcome text was in english. I think you can probably just make the caller always specify the language.